### ISSUE TITLE
SQL cells can't reference datasets by name

### ISSUE THREAD

**Issue by okennedy** [VizierDB/web-ui#45]
Given a table MOVIERATINGS...
```
select * from movieratings where action = 1;
```
running the cell triggers the error
```
Py4JJavaError:(u'An error occurred while calling t.createView.\n', JavaObject id=o106): Traceback (most recent call last):
  File "/usr/local/source/web-api/vizier/workflow/packages/userpackages/vizierpkg.py", line 435, in compute
    view_name = mimir._mimir.createView(mimir._jvmhelper.to_scala_seq(mimir_table_names), source)
  File "/opt/conda/envs/vizier/lib/python2.7/site-packages/py4j/java_gateway.py", line 1160, in __call__
    answer, self.gateway_client, self.target_id, self.name)
  File "/opt/conda/envs/vizier/lib/python2.7/site-packages/py4j/protocol.py", line 320, in get_return_value
    format(target_id, ".", name), value)
Py4JJavaError: An error occurred while calling t.createView.
: java.sql.SQLException: No such table or view 'MOVIERATINGS'
	at mimir.Database$$anonfun$table$1.apply(Database.scala:253)
	at mimir.Database$$anonfun$table$1.apply(Database.scala:249)
	at scala.Option.getOrElse(Option.scala:121)
	at mimir.Database.table(Database.scala:248)
	at mimir.sql.SqlToRA.convert(SqlToRA.scala:505)
	at mimir.sql.SqlToRA.convert(SqlToRA.scala:101)
	at mimir.sql.SqlToRA.convert(SqlToRA.scala:66)
	at mimir.sql.SqlToRA.convert(SqlToRA.scala:58)
	at mimir.Database.update(Database.scala:381)
	at mimir.MimirVizier$$anonfun$4.apply(MimirVizier.scala:365)
	at mimir.MimirVizier$$anonfun$4.apply(MimirVizier.scala:346)
	at mimir.MimirVizier$.logTime(MimirVizier.scala:1062)
	at mimir.MimirVizier$.createView(MimirVizier.scala:346)
	at sun.reflect.GeneratedMethodAccessor51.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
	at py4j.Gateway.invoke(Gateway.java:280)
	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
	at py4j.commands.CallCommand.execute(CallCommand.java:79)
	at py4j.GatewayConnection.run(GatewayConnection.java:214)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by okennedy** [comment]
Looks like this issue is due to tables not being referenced their standard names -- you need to use the query hints to get the table names.  Changing this to an enhancement request: People should be able to reference tables by their original names.

**Comment by mrb24** [comment]
fixed by commit: 4445ac5cf93d53c168d974c656812d0d7e4dd7f7
